src/gcp_scanner/scanner.py

```diff
@@ -435,14 +257,14 @@ def crawl_loop(initial_sa_tuples: List[Tuple[str, Credentials, List[str]]],
       if impers is not None and impers.get('impersonate', False) is True:
         iam_client = iam_client_for_credentials(credentials)
         if is_set(scan_config, 'iam_policy') is False:
-          iam_policy = crawl.get_iam_policy(
+          iam_policy = CrawlerFactory.create_crawler('iam_policy').crawl(
```

Comment:
We can add a short comment on why this crawler is used outside of new loop

Reply:
I moved the additional crawlers just under the new loop and added a section comment. Now I think it is more understandable why they are outside of the loop.